### PR TITLE
[Doc][Improve] Intro the multi-line text feature in document

### PR DESCRIPTION
**Pull request by nianhua99**
### Purpose of this pull request

#6682 

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/concept/config.md [hunk 1]
```diff
@@ -64,6 +64,18 @@ sink {
...
+
+In `hocon`, multiline strings are supported, which allows you to include extended passages of text without worrying about newline characters or special formatting. This is achieved by enclosing the text within triple quotes **`"""`** . For example:
+
+```
+var = """
+Apache SeaTunnel is a
+next-generation high-performance,
+distributed, massive data integration tool.
+"""
+```
```
Comment (Hisoka-X) [inline docs/en/concept/config.md:78]: How about add this case too?
```
sql = """ select * from "table" """
```

Comment (nianhua99) [reply docs/en/concept/config.md:78]: Sure, multiline text is indeed common in long SQL queries. I will add this case.

Comment (nianhua99) [reply docs/en/concept/config.md:78]: > How about add this case too?
> 
> ```
> sql = """ select * from "table" """
> ```

done

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @nianhua99

**Review by hailin0 — APPROVED** [review]

**Pull request merged**